woocommerce/woocommerce-android
# Enable Stripe Extension feature flag

Closes #5504


### Description


This PR enables support for IPP via Stripe Extension for all users. It also bumps the minimum required version of Stripe Extension to 6.2.0.

### Testing instructions

1. Verify "outdated plugin" screen is shown for versions < 6.2.0
2. Verify "outdated plugin" screen is NOT shown for version 6.2.0 - [woocommerce-gateway-stripe.zip](https://github.com/woocommerce/woocommerce-android/files/8080759/woocommerce-gateway-stripe.zip)

### Images/gif
